hack/run-locally.sh

```diff
@@ -239,7 +254,13 @@ echo "rebuilding the CNO"
 hack/build-go.sh
 
 # Autodetect the state of the cluster to determine which mode to run in
-if [[ -z "$(ls -A ${CLUSTER_DIR} 2> /dev/null | grep -v install-config.yaml | grep -v .openshift_install | grep -v env.sh)" ]]; then
+if [[ ! -z "${KUBECONFIG}" ]]; then
+    export CLUSTER_DIR=$PWD
```

Comment:
hm... what does this end up getting used for? Would it be better to use a temporary directory?

Reply:
This is used for the `${CLUSTER_DIR}/env.sh` which contains the image overrides. I could use a temporary directory. Would `/tmp/env.sh` be fine?

Reply:
Use `mktemp -d`.
Alternatively, write the environment to `${KUBECONFIG}.cno-env.sh` ?

Reply:
Used `mktemp -d`

Reply:
Acutally, I've been re-thinking this. Usually what we want is also to just create a `env.sh` and modify the image targeted, then re-deploy. 

I thus think it would be easier if we just use `CLUSTER_DIR=/tmp` if we only have a `$KUBECONFIG`. Then modify the `/tmp/env.sh` and re-deploy. I think it's easier as such.